LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/Regions/LoRaRegionEnum.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace LoRaTools.Regions
 {
-    public enum LoRaRegion
+    public enum LoRaRegionEnum
     {
```

Comment:
I don't think we should be naming an enum Enum. Can we call it LoRaRegion?

Reply:
Well, I changed the name due to the fact that there is another class named Region, and I always get mixed between the two, at least with this it is clear. Open for suggestion and/or best practices here.

Reply:
Not a big fan of calling enum XxxEnums too. Maybe LoRaRegionType.